Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.12.3
+
+RUN apk add --update --no-cache \
+  build-base \
+  nodejs-current \
+  curl \
+  npm \
+  bash \
+  libxslt \
```

Comment:
I don't think this is required.

Reply:
ah yes! leftover debugs. We need curl, bash, and maybe npm (unless it comes with nodejs-current). I'll check that last bit out before I commit its deletion.